airgun/entities/computeresource.py

```diff
@@ -59,6 +60,25 @@ def vm_poweroff(self, rhev_name, vm_name):
         if vm['Power'].widget.read() == 'On':
             vm['Actions'].widget.click(handle_alert=True)
 
+    def list_computeprofiles(self, entity_name, computeprofile_name):
+        view = self.navigate_to(self, 'Detail', rhev_name=entity_name)
+        view.compute_profiles.table.row(compute_profile=computeprofile_name)[
+            'Compute profile'].widget.click()
+        view.submit.click()
+
+    def edit_computeprofiles(self, values, entity_name, computeprofile_name):
```

Comment:
s/edit_computeprofiles/update_computeprofile
and pls accept `values` as the last positional arg for correspondence with all existing `update` methods - `(self, entity_name, computeprofile_name, values)`